### ISSUE TITLE
woadialer can't open in 1909v2,1910 and 1910-release

### ISSUE THREAD

**Issue by bihaiqiongtian** [WOA-Project/woadialer#24]
Hello,
I have tried it in the release of 1909v2,1910 and 1910-release,but woadialer can't open,It stays on the first page(the blue) and can't go on.

My windows system is 18875pro and 18908home-China,I tried a lot,but defeat.

When woadialer stays on the first page,There was no answer to the system call,unless I don't touch woadialer at the begining and never open to use it.

My English isn't very well,I hope you can understand what I say.

Please pay attention and solve it,thanks a lot!
                                                                                                               yours
                                                                                                                      Chinese fans

**Comment by DjArt** [comment]
Hi! Can you check few things:
1. Rebooting phone doesn't help?
2. Try to add any account with contacts, it may help.

**Comment by Simizfo** [comment]
hmmm, aren't those builds without cellular support?

**Comment by bihaiqiongtian** [comment]
Hello,
I have received your letter. Thank you very much for your attention. 
I have tested it:
1. The rebooting phone is useless if I can't open it initially. It's going to stay in blue.
2. To my surprise, when I reinstall the system of the final version(18362.387), and I tried to log in with my Microsoft account（before I used the offline account), everything became normal, I opened it successfully, it's awesome!
Then I tried again, it worked again! This problem was solved, although I don't know why.
Anyway, thanks for your reply. Best regard!
                                                                                                            Yours sincerely @DjArt
